### PR TITLE
framework/flamingo: add possibility to adapt SameSite property for cookies

### PR DESCRIPTION
**Pull request by Ompluscator**
Added possibility to adapt SameSite flag for cookies (so it can be supported in iFrame together with Secure=true flag).

### CODE REVIEW

File: framework/flamingo/sessions.go [hunk 1]
```diff
@@ -124,9 +127,18 @@ func (m *SessionModule) setSessionstoreOptions(options *sessions.Options) {
 	options.Domain = ""
 	options.Path = m.path
 	options.MaxAge = m.maxAge
-	options.Secure = m.secure
+	options.Secure = true
```
Comment (tessig) [inline framework/flamingo/sessions.go:130]: why do we need this to be always true?

File: framework/flamingo/sessions.go [hunk 2]
```diff
@@ -141,6 +153,7 @@ flamingo: session: {
 	cookie: {
 		secure: bool | *true
 		path: string | *"/"
+		sameSite: string | *"lax"
```
Comment (tessig) [inline framework/flamingo/sessions.go:156]:
```suggestion
		sameSite: *"lax" | "strict" | "none" | "default"
```

**Review by tessig — CHANGES_REQUESTED** [review]

File: framework/flamingo/sessions.go (hunk 1 above)
Comment (Ompluscator) [reply framework/flamingo/sessions.go:130]: @tessig  yup, you are right, it was pushed by mistake.

**Review by tessig — APPROVED** [review]
Sorry for the late reply. Looks fine and we will merge as soon as we solved our global nancy problem

**Pull request merged**